config/database.yml

```diff
@@ -1,25 +0,0 @@
-# SQLite version 3.x
-#   gem install sqlite3
-#
-#   Ensure the SQLite 3 gem is defined in your Gemfile
```

Comment:
For caching bug I used `autocomplete: 'off'` attribute and it seems a problem is solved.